modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="supporting-encapsulation_{context}"]
+= Supporting encapsulation
+
+As an OpenShift support engineer or developer, you want to inspect Kubernetes-specific mountpoints.
+
+== Locating the encapsulated namespaces
+
+Any monitoring tool that relies on the ability to run in the host OS and have visibility of mountpoints created by Kubelet, the Container Runtime, or containers themselves, will need to enter the container mount namespace in order to see these mountpoints. The provided `kubensenter` script will execute another command inside the Kubernetes mountpoint and can be used to adapt any existing tools.
```

Comment:
Where is the provided `kubensenter` script? Note we cannot link to external git repos in the docs.

Reply:
It's there already!  Every 4.12 (and later) OpenShift system will have the script pre-installed at `/usr/bin/kubensenter`, and this is true for both new deployments and upgrades to 4.12.

It's installed by the same RPM that installs the `kubelet` executable.

Reply:
As per Jim, it's installed with the product.